vmdb/app/controllers/provider_foreman_controller.rb

```diff
@@ -71,14 +71,23 @@ def refresh
   def provision
     assert_privileges("provider_foreman_provision")
     @prov_id = find_checked_items
-    @prov_id = params[:id] if @prov_id.empty?
-
-    render :update do |page|
-      page.redirect_to :controller     => "miq_request",
-                       :action         => "prov_edit",
-                       :prov_id        => @prov_id,
-                       :org_controller => "configured_system",
-                       :escape         => false
+    @prov_id.push(params[:id]) if @prov_id.empty?
+
+    provision_configured_systems = true
+    provision_configured_systems =
+      ConfiguredSystem.common_configuration_profiles_for_selected_configured_systems(@prov_id) if @prov_id.size > 1
```

Comment:
@kbrock Correct, if the configured system doesn't have any hostgroups available to it, no need to enter the dialog.

Also correct, ``` if ConfiguredSystem.common_configuration_profiles_for_selected_configured_systems(@prov_ids)```